### ISSUE TITLE
Shrine - set hint text to products to indicate what happens when you tap them (1)

### ISSUE THREAD

**Issue by rami-a** [material-components/material-components-flutter-gallery#483]
## Acceptance Criteria
* Semantic hint that indicates that tapping products will add them to cart

**Comment by perclasson** [comment]
This is already added for TalkBack for Android. For VoiceOver it does not read the onTapHint. Not sure if this is expected or not.

**Comment by perclasson** [comment]
I do not think this is intended behaviour and I could reproduce it in a simple case, created https://github.com/flutter/flutter/issues/49698 for this now. Marking this ticket as blocked for now.

**Comment by perclasson** [comment]
Found that this is intended behaviour:
https://api.flutter.dev/flutter/semantics/SemanticsHintOverrides-class.html

Will have to add a separate hint for VoiceOver.

**Comment by perclasson** [comment]
Since I am not sure that it is always a double tap that is the action, I'll modify it to 'Activate to add to cart'.
